plugins/woocommerce/src/RestApi/Routes/V4/OfflinePaymentMethods/Controller.php

```diff
@@ -2,7 +2,7 @@
 /**
```

Comment:
Small nitpick: I think the folder structure should be `RestApi/Routes/V4/Settings/OfflinePaymentMethods` to align with what @elazzabi  implemented in https://github.com/woocommerce/woocommerce/pull/61254, and you may need to update the namespacing for the PHP Classes accordingly. 🙂

Reply:
Thank you! I have moved it inside the Settings to align with the settings structure!